app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,34 @@
+class RecipesController < ApplicationController
+  def index
+    @recipes = current_user.recipes.where(user_id: params[:food_id])
+  end
+
+  def new
+    @recipe = current_user.recipes.new(user_id: params[:food_id])
+  end
+
+  def create
+    @recipe = current_user.foods.new(recipe_params)
```

Comment:
- [ ] Kindly reference an instance of recipes and not foods. Please correct that.

`current_user.recipes.new(recipe_params)`